### DISCUSSION TITLE
Recommended spell checker within Positron

### DISCUSSION THREAD

**Discussion by dmbates** [posit-dev/positron#5745] [Q&A]
Can anyone recommend a spell checker package to be used with Position, especially for Markdown or Quarto source files?

**Comment by nstrayer** [comment]
I personally am a fan of "Typos spell checker" https://open-vsx.org/extension/tekumara/typos-vscode It treats typos like syntax errors along with quick-fix options and everything!

**Comment by juliasilge** [comment]
We have some thoughts on spell check in https://github.com/posit-dev/positron/issues/1485, if you would like to add any additional details!
